rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -70,8 +76,19 @@ def getImageUploadAndColor():
     im = Image.open(file_path)
     im.show()
 
+    endPts = {}
```

Comment:
Can we rename this to something a little more intuitive like params?

Reply:
done